### PR TITLE
add bytesize SIG meta-omics

### PR DESCRIPTION
**Pull request by FranBonath**
@netlify /events/2024/bytesize_SIG_meta_omics

### CODE REVIEW

**Comment by FranBonath** [comment]
Please check the date, time, your affiliation and the content of the text :)

File: src/content/events/2024/bytesize_SIG_meta_omics.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+type: talk
+startDate: '2024-09-03'
+startTime: '13:00+02:00'
+endDate: '2024-09-03'
+endTime: '13:30+02:00'
+locations:
+  - links: https://kth-se.zoom.us/j/68390542812
+---
+
+In this weeks bytesize Daniel Lundin ([@erikrikarddaniel](https://github.com/erikrikarddaniel)), James Fellows Yates ([@jfy133](https://github.com/jfy133)) and Carson J. Miller ([@carsonjm](https://github.com/carsonjm)) introduce the [Special Interest Group "meta-omics"](https://nf-co.re/special-interest-groups/meta-omics). This group is a place to discuss the subject around _meta_ genomics, _meta_ taxonomics, _meta_ barcoding, _meta_ transcriptomics, _meta_ proteomics and all other _meta_ omics. Some pof the pipelines that would fall into this Special Interest Group are nf-core/ampliseq, nf-core/differentialabundance, nf-core/eager and many more.
```
Comment (maxulysse) [inline src/content/events/2024/bytesize_SIG_meta_omics.md:13]:
```suggestion
In this week's bytesize Daniel Lundin ([@erikrikarddaniel](https://github.com/erikrikarddaniel)), James Fellows Yates ([@jfy133](https://github.com/jfy133)) and Carson J. Miller ([@carsonjm](https://github.com/carsonjm)) introduce the [Special Interest Group "meta-omics"](https://nf-co.re/special-interest-groups/meta-omics). This group is a place to discuss the subject around _meta_ genomics, _meta_ taxonomics, _meta_ barcoding, _meta_ transcriptomics, _meta_ proteomics and all other _meta_ omics. Some of the pipelines that would fall into this Special Interest Group are nf-core/ampliseq, nf-core/differentialabundance, nf-core/eager and many more. 
```

Comment (erikrikarddaniel) [reply src/content/events/2024/bytesize_SIG_meta_omics.md:13]: In my opinion, the core pipelines should include at least Ampliseq, Mag, Metatdenovo and Magmap.

Comment (jfy133) [reply src/content/events/2024/bytesize_SIG_meta_omics.md:13]:
```suggestion
In this weeks bytesize Daniel Lundin ([@erikrikarddaniel](https://github.com/erikrikarddaniel)), James A. Fellows Yates ([@jfy133](https://github.com/jfy133)) and Carson J. Miller ([@carsonjm](https://github.com/carsonjm)) introduce the [Special Interest Group "meta-omics"](https://nf-co.re/special-interest-groups/meta-omics).

This group is a place to discuss the subject around _meta_ genomics, _meta_ taxonomics, _meta_ barcoding, _meta_ transcriptomics, _meta_ proteomics and all other _meta_ omics - basically any field that aim to study more than a single organism at the same time. Some of the pipelines that would fall into this Special Interest Group are nf-core/ampliseq, nf-core/taxprofiler, nf-core/mag, nf-core/metatdenovo, and nf-core/magmag, amongst others. 
```

**Review by jfy133 — COMMENTED** [review]
See my comment above

File: src/content/events/2024/bytesize_SIG_meta_omics.md (hunk 1 above)
Comment (erikrikarddaniel) [reply src/content/events/2024/bytesize_SIG_meta_omics.md:13]: I don't know how to suggest a correction on a suggestion but "magmag" should be "magmap".

Comment (jfy133) [reply src/content/events/2024/bytesize_SIG_meta_omics.md:13]: Fixed

**Comment by FranBonath** [comment]
@nf-core-bot fix linting please :)

**Pull request merged**